src/lib/components/CommunitySelectionModal/CommunitySelectionFooter.js

```diff
@@ -0,0 +1,28 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2022 CERN.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from 'react';
+import { Container, Segment } from 'semantic-ui-react';
+import { Trans } from '@translations/i18next';
+
+export const CommunitySelectionFooter = () => {
+  return (
+    <Container className="community-selection-footer">
+      <Segment textAlign="center">
+        <p>
+          <Trans i18nKey="Did not find a community that fits you? Upload without a community or <2> create your own.</2>">
+            Did not find a community that fits you? Upload without a community
+            or{' '}
+            <a href="/communities/new" target="_blank">
```

Comment:
Wondering if we should avoid a new tab here...
```suggestion
            <a href="/communities/new">
```
In any case, when you use `target=_blank`, you should always add:
` rel="noreferrer"`
https://stackoverflow.com/a/65178039/675178

Reply:
on it :+1: